src/types/applicant.ts

```diff
@@ -14,3 +14,37 @@ export type Applicant = {
   skills: Skill[];
   rating: number;
 };
+
+export function createApplicant(applicants: Applicant) {
+  let result = [];
+  if (Array.isArray(applicants)) {
```

Comment:
The if-check does not make sense, considering the input parameter is a single applicant

Reply:
That is why there's an if clause... to be able to convert both (a single applicant and multiple applicants) into the right format. I would leave it like that but i can also split it up into two function.